Remotion/Web/Core/UI/Controls/DropDownMenuImplementation/Rendering/DropDownMenuRenderer.cs

```diff
@@ -410,15 +410,17 @@ private void AppendMenuItem (DropDownMenuRenderingContext renderingContext, Stri
 
       string icon = GetIconUrl (renderingContext, menuItem, showIcon);
       string disabledIcon = GetDisabledIconUrl (renderingContext, menuItem, showIcon);
-      string text = showText ? "'" + menuItem.Text + "'" : "null";
-      string diagnosticMetadataText = showText ? menuItem.Text : "";
+      string text = showText
+          ? "'" + ScriptUtility.EscapeClientScript (menuItem.Text.ToString (WebStringEncoding.HtmlWithTransformedLineBreaks)) + "'"
```

Comment:
Another possible usecase for an overload. please ticket :)

Reply:
https://re-motion.atlassian.net/browse/RM-8172